c.html.markdown

```diff
@@ -63,26 +65,28 @@ enum days {SUN = 1, MON, TUE, WED = 99, THU, FRI, SAT};
 // libraries for the headers.
 // For your own headers, use double quotes instead of angle brackets, and
 // provide the path:
-#include "my_header.h" 		// local file
+#include "my_header.h" // local file
 #include "../my_lib/my_lib_header.h" //relative path
 
 // Declare function signatures in advance in a .h file, or at the top of
 // your .c file.
-void function_1();
-int function_2(void);
+void function_1(void);
+int function_2(int a, float b);
```

Comment:
I think it's fine the way it is and is easier to understand when it's simpler like this. I understand `()` is a compiler warning but in practice it doesn't matter if you don't even know how to access the variable length arguments of functions in C. I think mentioning `(void)` at the end is good but keeping it simple at the beginning is better.

```suggestion
void function_1();
int function_2(void);
```

Reply:
Even with `-Wall -Wextra` gcc (at least version 13.2.0 for me) compiles calling functions with `()` for the parameter list with some arbitrary parameters without any warning. clang does give me some warning. C23 did change this behavior though, making `(void)` and `()` equivalent parameter lists for function declarations, and I know gcc is on v14 now, so my OS's version is probably a bit out of date and this will not become as relevant for new C in not too long.

Having been burning by that unexpected behavior in the past, it was my intuition to include it when making these changes originally, but with the C23 changes putting it off to the end makes sense, or even leaving it out. I'll make some changes shortly